test/test_helper.rb

```diff
@@ -1,5 +1,3 @@
-lib_dir = File.dirname(__FILE__) + '/../lib'
-
+require 'bundler/setup'
 require 'minitest/autorun'
-$:.unshift lib_dir unless $:.include?(lib_dir)
-require 'debug_inspector'
+Bundler.require :default
```

Comment:
It's not required, but it acts as a shortcut for potentially many individual require's. It's not in the current diff anymore though: https://github.com/banister/debug_inspector/pull/16/commits/496ea5dad3697db64d984aa6075c3e00a1c33c64